src/index/mod.rs

```diff
@@ -71,6 +74,21 @@ impl Index {
     pub fn api(&self) -> &Api {
         &self.api
     }
+
+    pub fn run_git_gc(&self) {
+        let gc = Command::new("git")
+            .arg("-C")
+            .arg(&self.path)
+            .args(&["gc", "--auto"])
+            .output();
+        if let Err(err) = gc {
+            log::error!(
+                "Failed to run `git gc --auto`\nPath: {:#?}\nError:{:#?}",
```

Comment:
errors should be lowercase: https://rust-lang.github.io/api-guidelines/interoperability.html#error-types-are-meaningful-and-well-behaved-c-good-err
```suggestion
                "failed to run `git gc --auto`\npath: {:#?}\nerror: {:#?}",
```